tasks/main.yml

```diff
@@ -5,14 +5,14 @@
 - name: test to see if selinux is running
   command: getenforce
   register: sestatus
-  changed_when: false
-  ignore_errors: true
+  failed_when: false
+  always_run: true
```

Comment:
This is needed so the task runs even in check-mode. Otherwise the following tasks would fail. And since the "failed_when" option is set to false, it won't trigger a "changed".